python/rikai/spark/sql/codegen/mlflow_registry.py

```diff
@@ -224,7 +224,8 @@ def get_model_conf(self, spec, run):
 
     def get_options(self, spec, run):
         options = run.data.params
-        options.update(spec.get("options", {}))
+        if spec.get("options", {}):
```

Comment:
nit: why the change here?

Reply:
`spec` now is passed by jvm to python for the jdbc change, which is possible to be serialized as `{"options": None}` now. 

It was originally handled by the jvm code when passing through `py4j` to change `None` to `{}`.